rust_snuba/src/processors/spans.rs

```diff
@@ -216,21 +222,6 @@ impl TryFrom<FromSpanMessage> for Span {
             .map(|(k, v)| (k, v.value))
             .unzip();
 
-        if let Some(http_method) = from.sentry_tags.http_method {
```

Comment:
These are not meant to be in `tags`, just `sentry_tags`, the Python processor was wrong.